### PR TITLE
Sync with internal repo through rebase

### PR DESCRIPTION
**Pull request by catdollar**
pickup latest internal gdmix-trainer and gdmix-data code

### CODE REVIEW

**Review by junshi15 — APPROVED** [review]
LGTM

File: gdmix-trainer/src/gdmix/util/constants.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 OFFSET = "offset"
 SAMPLE_ID = "sample_id"
 SAMPLE_WEIGHT = "sample_weight"
-FEATURE_BAGS = "feature_bags"
+FEATURE_BAGS = "feature_bag"
```
Comment (wordpr3ss) [inline gdmix-trainer/src/gdmix/util/constants.py]: better make the name and value consistent as everything else.

Comment (jakiejj) [inline gdmix-trainer/src/gdmix/util/constants.py]: Change the constant name for consistency and bump the version?

Comment (catdollar) [reply gdmix-trainer/src/gdmix/util/constants.py]: I think the file you reviewed is outdated, the latest code is `FEATURE_BAG = "feature_bag"`.

**Pull request merged**